### ISSUE TITLE
OLMo-2 held-out validation data

### ISSUE THREAD

**Issue by chawins** [allenai/OLMo#755]
### ❓ The question

I'm very excited about the release of OLMo-2, and thank you for doing all the hard work! 

I noticed that the [config file](https://github.com/allenai/OLMo/blob/main/configs/official-1124/OLMo2-7B-stage1.yaml#L102) of OLMo-2 7B and 8B has commented out the validation set used in the first OLMo. Why is this the case? All those data (along with Paloma?) were no longer held out and used in the pre-training?

**Comment by dirkgr** [comment]
By and large we found that we never looked at those numbers during a training run, and so we didn't want to pay the time penalty of running that evaluation all the time.

In retrospect, maybe we should have had at least some of the Paloma sets in there. This can be computed after the fact, but it's a little bit cumbersome.

**Comment by dirkgr** [comment]
To answer your second question, no, we did not decontaminate against Paloma this time.

**Comment by chawins** [comment]
Thank you for the answer and also the extra wisdom!
I guess that the perplexity on validation set is not main signal you're looking to optimize out of the pre-trained models. So not paying attention to Paloma is a reasonable decision to me also.

I'm doing some research in privacy and memorization of LLMs where it turns out that these IID unseen held-out data are actually quite valuable. No other models AFAIK (apart from Pythia) have a held-out set like OLMo + Paloma.
